nh_eobs_ldap/tests/res_users/test_change_password.py

```diff
@@ -0,0 +1,22 @@
+from openerp.tests.common import TransactionCase
+from openerp.osv.osv import except_osv
+
+
+class TestResUsersChangePassword(TransactionCase):
+    """
+    Test that the change_password method override on the res.users model has
+    been applied successfully
+    """
+
+    def setUp(self):
+        super(TestResUsersChangePassword, self).setUp()
+        user_model = self.env['res.users']
+        self.user = user_model.browse(self.uid)
+
+    def test_raises_on_changing_ad_password(self):
+        self.user.write({'ldap_authenticated': True})
```

Comment:
Can use V8 style which is `self.user.ldap_authenticated = True`.

Reply:
👍